tests/VObject/ITip/BrokerProcessReplyTest.php

```diff
@@ -373,6 +440,126 @@ public function testReplyNewExceptionTz(): void
         $this->process($itip, $old, $expected);
     }
 
+    public function testReplyNewExceptionRecurrenceIdInDifferentTz(): void
+    {
+        // This is a reply to 1 instance of a recurring event. This should
+        // automatically create an exception.
```

Comment:
```suggestion
        // This is a reply to 1 instance of a recurring event.
```

I don't understand why this comment had "This should automatically create an exception."

Reply:
It is a copy/paste from all existing `testReplyNewException...` test cases.
The original event didn't had the exception, but the attendee replies to one recurrence only. So the reply should create the exception event.

Reply:
My brain was thinking "exception" as in "throw an exception"!
But in this context, "exception event" means a recurrence of a recurring event that has "something different" about it. In this case, it has an attendee who is attending that particular recurrence of the event.

All good.